src/views/Namespaces/Namespace/Edit.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="scrollable">
+    <div class="container">
+      <h1 v-if="isEdit" class="text-center mt-4">{{ $t('namespace.edit', {name: namespace.name}) }}</h1>
+      <h1 v-else class="text-center mt-4">{{ $('namespace.create') }}</h1>
```

Comment:
how did this slip by?

Reply:
By this you mean the missing 't' right ? Or am i missing something else?